### ISSUE TITLE
This doesn't seem to work with large files

### ISSUE THREAD

**Issue by ugultopu** [soulshined/vscode-jump-to-character-position#6]
I have a 163 MB file and it doesn't work on it. On smaller files, it works.

**Comment by soulshined** [comment]
Hi @ugultopu thanks for taking the time to give feedback. Sadly, this is a limitation of vscode's extension ecosystem itself and nothing we can do about it :/ 

I've realized this hasn't been a direct issue logged here but instead discussion happened by way of an extension review 
https://marketplace.visualstudio.com/items?itemName=davidfreer.go-to-character-position&ssr=false#review-details so there's no documentation around it in my github issues. Here's the initial review:

> On a Mac, I have mapped the keystroke and it works fine on a small file.
>
> But I have a massive JSON file I'm trying to work with.  Pressing the keystroke when in that file, nothing happens.
>
> The file is (temporarily) here: https://drive.google.com/file/d/10goTi6nntxs0-1r0f20DhiiwjmWeEGmj/view?usp=sharing

After some initial review and analysis from myself - I then [opened an issue](https://github.com/microsoft/vscode/issues/146784) myself. The answer I got was the following: https://github.com/microsoft/vscode/issues/146784#issuecomment-1089253422

So in short, vscode limits extension syncing for large documents (large being greater than 50mb). This is true for all extensions

In another [unrelated issue](https://github.com/soulshined/vscode-jump-to-character-position/issues/4), as a by-product a feature request was created to support this extension's functionality natively, in which I have posted some feedback on some limitations, like this one you ran into, that impacts navigation based extensions: https://github.com/microsoft/vscode/issues/196067#issuecomment-1783807657

I would encourage those interested in supporting jump to character natively to provide feedback there and watch progress as the vscode team navigates this
